backend-shared/plannershared.cpp

```diff
@@ -301,3 +309,9 @@ void plannerShared::set_display_variations(bool value)
 {
 	DivePlannerPointsModel::instance()->setDisplayVariations(value);
 }
+
+// Handle when user change length measurement type
+void plannerShared::unit_lengthChangedSlot(int value)
+{
+	qDebug() << "---> change METER <-> FEET";
```

Comment:
I don't think we want to check this into production code...

Reply:
It was just there to prove that the signal got received, I had some serious problems with qPrefUnit. 

Gone